### ISSUE TITLE
Extensions are broken in 2024.1.6

### ISSUE THREAD

**Issue by EotT123** [manifold-systems/manifold#555]
**Describe the bug**
Extensions aren't working anymore. Compiling the sample projects result in all kinds of errors. With the previous version (2024.1.5), no such errors occur.

**Desktop (please complete the following information):**
- OS Type & Version: Windows 10
- Java/JDK version: openjdk 21
- IDE version (IntelliJ IDEA or Android Studio): IntelliJ IDEA 2023.3.4 (Community Edition)
- Manifold version: 2024.1.6 (latest version at the moment)
- Manifold IntelliJ plugin version: 2023.3.16 (latest version at the moment)

**Comment by rsmckinney** [comment]
Latest IJ plugin is 2023.3.17.  It should be downloadable from IJ plugins settings.

**Comment by EotT123** [comment]
But it even doesn't work when building with maven. The plugin isn't needed in that case?

**Comment by rsmckinney** [comment]
All the tests build with maven and most of the sample projects. Can I access your project to see what is going on? Or is there a minimal project you can provide to reproduce?

**Comment by EotT123** [comment]
I was also coming to the conclusion that it was not related to the extensions. Thanks for looking into it.

**Comment by rsmckinney** [comment]
Fix available with release 2024.1.7.  IJ plugin also in the queue pending JB approval, sb available before EOD tomorrow (3/18)

**Comment by EotT123** [comment]
It's not fixed for me (both in intellij using the new plugin version as building with maven). 

FYI: You can setup github workflows to build the project and run the tests. This can be configured to be executed with all the java versions you want (and automatically on each commit/PR merge/etc.). This way you will directly know if something is broken.

**Comment by rsmckinney** [comment]
Are you sure you’re using latest? I can build and test fine with all JDKs + all sample projects

**Comment by EotT123** [comment]
Yes. To be completely sure, I removed my maven repository, and only version 2024.1.7 is redownloaded afterwards. 
I've tested with a few projects, they all fail.

I've update maven to the latest version (3.9.6) and OpenJDK to latest version (21.0.2).
I've also tried using Oracle & Eclipse Temurin java implementations, but no difference.

**Comment by rsmckinney** [comment]
Found the problem. Sorry for the trouble.  Indeed, manifold testing across JDKs needs help. I should have this fixed and ready for use by tomorrow, same drill.  Again, apologies for this mess and thanks for your patience.

**Comment by rsmckinney** [comment]
As a sanity check on 2024.1.7 I retested with a clean environment and I was wrong about it not working with JDK 21, it _does_ work (for tests and sample projects on all jdks). So I'm still a bit confused as to what is wrong on your end.

In any case I just made some changes that may address the issues you are experiencing, but I can build and test successfully with or without them everywhere. Would help a lot if you could provide a sample project or instructions or a build file help me reproduce the problem. Thanks!

Later tonight I'll commit the changes I've made and make a new release. I'm hoping it clears things up for you.

**Comment by EotT123** [comment]
Still not working. I'll try to narrow it down and make a small sample project.

**Comment by EotT123** [comment]
I narrowed it down a bit: It fails when manifold is used together with Lombok. I'm going to narrow it down further to see if it is due to something specific, or just the combination. 
It used to work in the past,

**Comment by EotT123** [comment]
Just adding this to the annotationProcessorPaths makes the build fail.
```
<path>
	<groupId>org.projectlombok</groupId>
	<artifactId>lombok</artifactId>
	<version>1.18.30</version>
</path>
```

I've tested it with the `manifold-sql-sample-project` project. Without the Lombok annotation processor, it compiles, no errors. But with it, I get errors.

**Comment by rsmckinney** [comment]
Thanks!  Others have been reporting this as well. I have a local fix for this (finally!) that will be available in a release later tonight.

**Comment by rsmckinney** [comment]
Fix available with release 2024.1.9. Tested w/wo lombok

**Comment by EotT123** [comment]
I confirm that it works again. Thank you for your continuous efforts in finding a solution!
